junifer/utils/logging.py

```diff
@@ -297,11 +351,19 @@ def configure_logging(
         #     "%(asctime)s [%(levelname)s] %(message)s "
         #     "(%(filename)s:%(lineno)s)"
         # )
-    formatter = logging.Formatter(fmt=output_format)
+    formatter = ColorFormatter(fmt=output_format)
```

Comment:
What about an option for a `--color` parameter for switching between the formatters? Maybe some remote machine doesn't have xterm support and that might break it?

Reply:
I will auto-detect color support